tests/test_blob_ndjson_format.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import json
+from azure.storage.blob import BlobServiceClient
+from azure.core.exceptions import ResourceNotFoundError
+
+
+class TestAzureBlobNDJSONFormat(unittest.TestCase):
```

Comment:
The name of the module can be  azblob_ndjson_validation.py and similarly s3_ndjson_validation.py

Reply:
done